network/alsp/model/params.go

```diff
@@ -45,3 +45,9 @@ const (
 	// speed is 1, and it takes around a day to recover from each disallow-listing.
 	InitialDecaySpeed = 1000 // (Don't change this value)
 )
+
+type PenaltyParams struct {
+	DisallowListingThreshold float64
+	DefaultPenaltyValue      float64
+	InitialDecaySpeed        int
+}
```

Comment:
```suggestion
```

This struct is unused

Reply:
removed 52d335a61a183b079e46842b7efb70c333482123